sites/all/modules/custom/lesson_batch/lesson_batch.pages.inc

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @file
+ * Renders output data.
+ */
+
+/**
+ * Page callback: Displays a a table with uploaded files.
+ *
+ * @see lesson_batch_menu().
+ */
+function lesson_batch_landing_table() {
+  drupal_add_js(drupal_get_path('module', 'lesson_batch') . '/js/lesson_batch.js');
+  drupal_add_js(array('uid' => $GLOBALS['user']->uid, ), 'setting');
```

Comment:
The best way to work with global User object is
global $user;
$a = $user->uid;